build_debian.sh

```diff
@@ -246,7 +246,10 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     mcelog                  \
     ndisc6                  \
     python-pip              \
-    ipmitool
+    ipmitool                \
+    lsof                    \
+    sysstats                \
```

Comment:
It the package name issue? sysstat or sysstats?

Reply:
Thanks @xumia